### PR TITLE
Add org.gnome.World.Secrets

### PR DESCRIPTION
**Pull request by A6GibKm**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

---

For the GTK 4 port of [Password Safe](https://flathub.org/apps/details/org.gnome.PasswordSafe), we renamed the app to Secrets and changed the application ID. With @okias we plan to deprecate the old flatpak after a beta phase in a week or so.

### CODE REVIEW

**Comment by okias** [comment]
I can confirm. Signing this request.

**Comment by A6GibKm** [comment]
bot, build org.gnome.World.Secrets

**Comment by A6GibKm** [comment]
bot, build org.gnome.World.Secrets

File: org.gnome.World.Secrets.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/cmake",
+        "/lib/debug",
```
Comment (hfiguiere) [inline org.gnome.World.Secrets.json:18]:
```suggestion
```
it's actually needed and will be in a separate flatpak.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.gnome.World.Secrets.json (hunk 1 above)
Comment (A6GibKm) [reply org.gnome.World.Secrets.json]: Thanks, didn't know.

**Comment by A6GibKm** [comment]
bot, build org.gnome.World.Secrets

**Comment by A6GibKm** [comment]
bot, build org.gnome.World.Secrets

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request closed without merging**